README.md

```diff
@@ -1,2 +1,3 @@
 # Documentation-page
-Freecodecamp Technical Documentation page
+
+Freecodecamp Technical Documentation page.
```

Comment:
It will be better if you used a descriptive readme. md in your project. This will guide any user of your project on what you are doing.